Gruntfile.js

```diff
@@ -41,6 +41,22 @@ module.exports = function(grunt) {
             mixinPrefix: ''
           }
         }
+      },
+
+      // create a helper css file for BiDi-proof icons
+      bidi: {
```

Comment:
Here I feel like I'm abusing grunt-webfont... Do you see another better way of doing this?